lib/glific/templates/interactive_templates.ex

```diff
@@ -113,17 +113,61 @@ defmodule Glific.Templates.InteractiveTemplates do
   @spec create_interactive_template(map()) ::
           {:ok, InteractiveTemplate.t()} | {:error, Ecto.Changeset.t()}
   def create_interactive_template(attrs) do
```

Comment:
Markdown check doesn't exist while updating interactive template, shouldn't we add there too?, since users can always add markdown later too.